api/src/main/java/org/openmrs/api/db/ConditionDAO.java

```diff
@@ -13,6 +13,7 @@
 
 import org.openmrs.Condition;
 import org.openmrs.Patient;
+import org.openmrs.api.ConditionService;
```

Comment:
do we need this import here ??

Reply:
Same as the above!

Reply:
Do you get a compiler error when you remove the import?

Reply:
No, I don't get any compiler error. Do you want me to remove this ?